lib/Alien/Base/ModuleBuild.pm

```diff
@@ -465,14 +470,15 @@ sub ACTION_alien_install {
   $self->alien_refresh_manual_pkgconfig( $self->alien_library_destination );
   $self->config_data( 'finished_installing' => 1 );
 
-  # to refresh config_data
-  $self->SUPER::ACTION_config_data;
-
   if ( $self->notes( 'alien_blib_scheme') || $self->alien_stage_install) {
-    # reinstall config_data to blib
-    $self->process_files_by_extension('pm');
+    ## reinstall config_data to blib
+    #$self->process_files_by_extension('pm');
```

Comment:
I don't really like having an empty `if` body, but if it makes the code easier to understand, we should keep it.